ospf6d/ospf6_auth_trailer.c

```diff
@@ -0,0 +1,817 @@
+/*
+ * Copyright (C) 2021 Abhinay Ramesh
+ *
+ * This file is part of GNU Zebra.
+ *
+ * GNU Zebra is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation; either version 2, or (at your option) any
+ * later version.
+ *
+ * GNU Zebra is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; see the file COPYING; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA
+ */
+
+#include "memory.h"
+#include "zebra.h"
```

Comment:
Again: `zebra.h` must be the first include.

Reply:
I am changing it. But is this a constrain ? Does all files in FRR follow this ?

Reply:
Yes it is, there is also a script to verify all files in `python/firstheader.py`. If the header is not first, it can break autoconf / configure in very confusing ways.